frontend/src/util/types/Requisition.ts

```diff
@@ -0,0 +1,139 @@
+import {User} from "./User";
+import {Project} from "./Project";
+import {Vendor} from "./Vendor";
+import {gql} from "@apollo/client";
+import {SemanticCOLORS} from "semantic-ui-react/dist/commonjs/generic";
+
+export type RequisitionStatus =
+    "DRAFT" |
+    "SUBMITTED" |
+    "PENDING_CHANGES" |
+    "READY_TO_ORDER" |
+    "ORDERED" |
+    "RECEIVED" |
+    "CANCELLED";
+
+export const StatusToColor = (status: RequisitionStatus): SemanticCOLORS => {
+    switch (status) {
+        case "DRAFT": return "grey"
+        case "SUBMITTED": return "yellow"
+        case "PENDING_CHANGES": return "orange"
+        case "READY_TO_ORDER": return "green"
+        case "ORDERED": return "blue"
+        case "RECEIVED": return "blue"
+        case "CANCELLED": return "red"
+    }
+}
+
+export const StatusToString = (status: RequisitionStatus): string => {
+    switch (status) {
+        case "DRAFT": return "Draft"
+        case "SUBMITTED": return "Submitted"
+        case "PENDING_CHANGES": return "Pending Changes"
+        case "READY_TO_ORDER": return "Ready to Order"
+        case "ORDERED": return "Ordered"
+        case "RECEIVED": return "Received"
+        case "CANCELLED": return "Cancelled"
+    }
+}
+
+export type Requisition = {
+    id: number,
+    headline: string,
+    description: string,
+    status: RequisitionStatus,
+    pointOfContact: User,
+    createdBy: User,
+    project: Project,
+    vendor: Vendor,
+    projectRequisitionId: number,
+    paymentRequiredBy: Date,
+    requisitionitemSet: RequisitionItem[],
+    referenceString: string,
+    canEdit: boolean,
+    otherFees: number
+}
+
+// Sums up total costs for requisition items
+export const getTotalItemsCost = (requisition: Requisition): string => {
+    let total = 0;
+    for (const requisitionItem of requisition.requisitionitemSet) {
+        total += requisitionItem.quantity * requisitionItem.unitPrice
+    }
+
+    return total.toFixed(2);
```

Comment:
`toFixed` is returning a string; is that what we want?  I think it might make more sense for this function return a number (from what I have read - [thing 1](https://www.syspro.com/blog/applying-and-operating-erp/rounding-for-accounting-accuracy/), [thing 2](https://stackoverflow.com/questions/3840793/rounding-standards-financial-calculations), there is no strong standard for when to round) and let the consumer of the result decide how to display it, since it might want to add up the total costs of many requisitions